scripts/index.js

```diff
@@ -0,0 +1,118 @@
+
+
+
+/// city changing logic for button and enter key 
+
+const changeCityButton = document.querySelector('#changeCityButton');
+const inputBox = document.querySelector('#cityNameInput')
+
+const updateCityName = () => {
+    const cityNameInput = document.getElementById("cityNameInput").value;
+    const headCityName = document.querySelector("#headCityName");
```

Comment:
I'd recommend standardizing on `getElementById` or `querySelector` and not mixing them.